### DISCUSSION TITLE
'The GitHub Student Developer Pack' was not verified in spite of my student ID is valid

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53463] [GitHub Education] [answered]
I submitted the application for 'The GitHub Student Developer Pack' with valid my student ID.
But, this application was rejected.

The message in e-mail of rejection is below.
```
・You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
・You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
```

I'm Japanese student, so my student ID is written in Japanese.
I think the reason of rejection is the OCR system of valification is not compatible with Japanese language.

I need any information to solve this issue!

**Comment by samus-aran** [comment, marked as answer]
Hi @snowball-rain 👋 

> I think the reason of rejection is the OCR system of valification is not compatible with Japanese language.

Good assumption. 👍 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by ghost** [reply]
Hi @samus-aran 

Thank you for your reply!
Your information is very helpful!
My application of 'The GitHub Student Developer Pack' was passed! Thanks!

My procedure is below. I hope this information is helpful for someone.
---
1. Set GitHub profile.
Username: Nickname
Name : Real name in English
bio:Name of University in English(This name is same as Github Education application form)
2. Scan my Student ID(Written in Japanese)
3. Crop my Student ID in the scan image.
4. Edit my Student ID's scan image to write English translation.
Write the translation in red text next to the original Japanese text.
3 informations should be translated.
- Name of University(in English)
- Real Name(in English)
- Valid Date (in English)(example: "valid until 04/2023")
5. Display the translated Student ID in the PC Monitor.
6. Open "The GitHub Student Developer Pack"'s upload page using Smartphone.
7. Take photo of the PC Monitor(it's displayed Student ID) using Smartphone.
note: In my case, image file uploading was restricted. Camera mode is only allowed.
8. Submit.
9. The application was passed!

**Reply by TalhaKarakoyunlu** [reply]
More than a year after, this message helped me out a lot. Thank you very much, Sato-san!

**Comment by kibo1no1shirei-kan** [comment]
Hello do i really need to use my real name as my user name or even in my bio i dont like it if i do

**Reply by samus-aran** [reply]
Hi @kibo1no1shirei-kan 👋 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.
We are extremely proud that the GitHub Global Campus has provided _millions_ of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 

However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification.

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily comparing the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information as it's only required upon each new submission. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

**Comment by kibo1no1shirei-kan** [comment]
Thank you however I do have a question regarding schools since my actual school is Victory Christian School its not appear on the map so I decided to use homeschool global Philipines instead which is now the front says am from Victory Christian but i think the orc would 
appreciate it if I am able to provide the back which contained am part of the homeschool global community.

**Reply by samus-aran** [reply]
Hi @kibo1no1shirei-kan  👋

We’re happy you’re here but let’s not hijack this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Reply by ZainabHaiderMalik** [reply]
> Thank you however I do have a question regarding schools since my actual school is Victory Christian School its not appear on the map so I decided to use homeschool global Philipines instead which is now the front says am from Victory Christian but i think the orc would 
> appreciate it if I am able to provide the back which contained am part of the homeschool global community.
**Did you find a solution to this problem? 
**

**Comment by ZainabHaiderMalik** [comment]
****Hello, I did everything I could to prove the case, but the refusal came until I went to the university website and photographed from inside the college with proofs of affiliation, but to no avail... Note that we are a college located in a different location from the university, but we use the campus mail, the last response was this👇 **
**
````

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jul 7, 2023 for @ZainabHaiderMalik.

Attendees of this school must verify and use their academic email address.

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.

Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?

The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket.


Once you have resolved the issues with your sumbission, visit us again and reapply.

Have questions? Join the GitHub Education Community for answers.

💖 The GitHub Education Team``

**Reply by kibo1no1shirei-kan** [reply]
hello my id has needed two images the back is the homeschool global part while the front is my identity, and am homeschooled how do i indicate that am homeschooled? also, it seems like uploading two photos on the website is not possible pls test this.

**Comment by ParaN3xus** [comment]
Thank you! My verification has passed!

**Comment by ZainabHaiderMalik** [comment]
How did it work for you? Tell me I'm not near campus?

في الأحد، ١٠ أيلول ٢٠٢٣, ١١:١٠ ص ParaN3xus ***@***.***> كتب:

**Reply by ParaN3xus** [reply]
Then you should go to your school and apply again

**Comment by ZainabHaiderMalik** [comment]
We're on vacation now, what did you do to check
؟

في الأحد، ١٠ أيلول ٢٠٢٣, ١:١٢ م ParaN3xus ***@***.***> كتب:

**Reply by ParaN3xus** [reply]
Wait until you're back, or try something like fakegps.

**Comment by ZainabHaiderMalik** [comment]
Ok, I will.

في الأحد، ١٠ أيلول ٢٠٢٣, ١:١٨ م ParaN3xus ***@***.***> كتب:

**Comment by ZainabHaiderMalik** [comment]
I'm still stuck. 😄

في الجمعة، ٣ أيار ٢٠٢٤، ٣:٢٨ ص TALHA BAHADIR KARAKOYUNLU <
***@***.***> كتب:
